src/RoleObjects/SpinButtonData.js

```diff
@@ -15,6 +15,10 @@ export default class SpinButtonData extends RangeData {
     this._reactProps.onKeyDown = this._onKeyDown; // the keydown listener is needed for this role to function per WAI-ARIA practices even when using a semantic tag/attribute
   }
 
+  get enableKeyEvents() {
+    return super.enableKeyEvents;
+  }
```

Comment:
The corresponding setter was already in this class (directly above this change), so the getter missing was an oversight when the spin button role was implemented.